### PR TITLE
Governable

### PR DESCRIPTION
**Pull request by nkuba**
Instead of using Open Zeppelin's Ownable implementation, we switch to
Governable. Which is a simpler and smaller contract that is upgrade-safe.
Also, this approach is used in other projects
@threshold-network/solidity-contract, @keep-network/random-beacon, and
@keep-network/ecdsa.

Refs https://github.com/keep-network/keep-core/pull/2921

### CODE REVIEW

File: solidity/deploy/08_transfer_governance.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```
Comment (pdyraga) [inline solidity/deploy/08_transfer_governance.ts:4]: Should we use this function in deployment scripts now?

**Review by pdyraga — COMMENTED** [review]
Just one question @nkuba, everything else looks good to me.

File: solidity/deploy/08_transfer_governance.ts (hunk 1 above)
Comment (nkuba) [reply solidity/deploy/08_transfer_governance.ts:4]: I'm sorry, I don't get your question. 

We used to transfer the ownership, now we transfer the governance. It's just a function name change, the result is the same.

Comment (pdyraga) [reply solidity/deploy/08_transfer_governance.ts:4]: My fault, never mind. 🤦🏻

**Review by pdyraga — APPROVED** [review]

**Pull request merged**